content/finance/spending-company-money.md

```diff
@@ -42,26 +42,54 @@ The estimated spend for interns is **$925\*** and includes only the electronic i
 
 ## Computers
 
-Once your contract is signed, you will need to purchase and expense your Sourcegraph computer. Computers and laptop insurance (this includes Apple Care if you purchase a Mac as your work computer) are fully reimbursable outside of the [desk setup total estimate above](#desk-set-up), however, engineering roles have different hardware requirements from other roles (please see below for necessary specs corresponding to your role). Computers are considered property of the company.
+Once your contract is signed, you will need to get your Sourcegraph computer. Computers are considered property of the company. Please see below how to get your computer and which specs to get depending on your role.
 
-**Laptop (Engineering team, Customer Engineering team and Customer Support team):**
+**US based teammates**
+US base teammates need to place their order via [this form](https://forms.gle/1PVxXwLJ9WFJvref9) and our Tech Ops team will purchase them through CDW. (Please note there is nothing we can do about wait times and will update you when the order has been placed and when we have any more information about shipping dates).
 
-- 16" MacBook Pro, 2.4 GHz (5 GHz turbo) 8‑core (upgrade), 64 GB (upgrade), 1 TB SSD
+**Non US teammates**
+Non US teammates will be responsible for purchasing and expensing their Sourcegraph computer. Computers and laptop insurance (this includes Apple Care if you purchase a Mac as your work computer) are fully reimbursable outside of the [desk setup total estimate above](#desk-set-up). You can find information on how to expense computers [here](expenses.md).
+
+Different specs are required repending on your role, please see below for necessary specs corresponding to your role.
+
+### Customer Engineering team laptops:
+
+- Apple MacBook Pro - 14.2" M1 Pro - 16 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-14.2-m1-pro-16-gb-ram-1-tb-ssd-us/6738975?enkwrd=6738975)
+- Apple MacBook Pro - 16.2" M1 Pro - 16 GB RAM, 1 TB SSD
+  - [See product in CDW for reference](https://www.cdw.com/product/apple-macbook-pro-16.2-m1-pro-16-gb-ram-1-tb-ssd-us/6738957?enkwrd=6738957)
 - Any other laptop that is of equal or lesser value than the pre-approved laptop.
 
-**Laptop (product team):**
+### Product team laptops:
 
-- 13" MacBook Pro, 2.3 GHz (4.1 GHz turbo) 4-core (upgrade), 32 GB (upgrade), 512 GB SSD. If you need to update any specs, please reach out to @sourcegraph/people-ops for approval.
+- Apple MacBook Pro - 16.2" M1 Max - 32 GB RAM, 512 GB SSD
+  - [See product in CDW for reference]()
 - Any other laptop that is of equal or lesser value than the pre-approved laptop.
 
-**Laptop (all other teams):**
+### Engineering team laptops:
 
-- 13" MacBook Pro, 2.0 GHz (3.8 GHz turbo) Intel Core i5 processor, 16 GB, 512 GB SSD. If you need to update any specs, please reach out to @sourcegraph/people-ops for approval.
+- Apple MacBook Pro - 16.2" M1 Max - 64 GB RAM, 1 TB SSD
```

Comment:
@InesRoitman should this one be 14.2" (both this one and the one below are 16.2" right now, and the only  difference is RAM)?

Reply:
We are still defining some stuff with Tech Ops. I believe these should be 13'' and 14'', so I've asked them.

I think this PR should have been a draft PR as this is not the final take.

Reply:
Sure, I wanted to see the current recommendation and I couldn't understand if this one or the one below should be picked - thanks!